modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -72,6 +72,13 @@ func TestMsgRegisterInterchainAccountValidateBasic(t *testing.T) {
 			},
 			false,
 		},
+		{
+			"order is not valid",
+			func() {
+				msg.Order = channeltypes.NONE
```

Comment:
Small nit I just realised: in the protos we called this field `order` for `MsgRegisterInterchainAccount, while in `MsgChannelOpenInit` is called `ordering`. Should we rename `order` to `ordering` for consistency?

Reply:
yea, think we should!

Reply:
Yeah, go for it!

Reply:
ah, would be nice to backport that change and just saw about BP this PR. can open as separate instead to have it backport cleanly